AIDevGallery/Controls/SampleContainer.xaml.cs

```diff
@@ -46,6 +45,7 @@ public List<string> NugetPackageReferences
     private CancellationTokenSource? _sampleLoadingCts;
     private TaskCompletionSource? _sampleLoadedCompletionSource;
     private double _codePaneWidth;
+    private ModelType _wcrApi;
```

Comment:
Shouldn't this be nullable? Its not used elsewhere if not on a WCR API, but it might in the future, so its safer to be explicit on typing here.